tests/config_test.py

```diff
@@ -702,6 +705,200 @@ def teardown_class(cls):
         dbconnector.load_namespace_config()
 
 
+class TestBMPConfig(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        os.environ['UTILITIES_UNIT_TESTING'] = "1"
+        yield
+        print("TEARDOWN")
+        os.environ["UTILITIES_UNIT_TESTING"] = "0"
+
+    def test_disable_bgp_neighbor_table(
```

Comment:
>def test_disable_bgp_neighbor_table [](http://example.com/codeflow?start=4&length=35)

Minor suggestion: use @pytest.mark.parametrize to minimize repeating code.

Reply:
Thanks for the suggestion, updated.